cms/tests/test_admin.py

```diff
@@ -295,56 +295,6 @@ def test_change_template(self):
             )
             self.assertEqual(response.status_code, 200)
 
-    def test_changelist_items(self):
```

Comment:
What is the justification for removing the two tests?

Reply:
About `test_changelist_items`, we did not pass anymore `items` into context, so this test is no more relevant.
Then concerning `test_changelist_get_results`, this is the same, we did get node through a query into changelist.